components/ActionSheet/examples/ActionSheetTypicalUseExampleViewController.m

```diff
@@ -36,40 +36,37 @@ - (instancetype)init {
     _colorScheme =
         [[MDCSemanticColorScheme alloc] initWithDefaults:MDCColorSchemeDefaultsMaterial201804];
     _typographyScheme = [[MDCTypographyScheme alloc] init];
+    _containerScheme = [[MDCContainerScheme alloc] init];
```

Comment:
Please remove the Color and Typography scheme properties and use only the container scheme.

Reply:
Done